hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/metadata/SCMMetadataStoreImpl.java

```diff
@@ -313,6 +330,20 @@ private void checkTableStatus(Table table, String name) throws IOException {
       LOG.error(String.format(logMessage, name));
       throw new IOException(String.format(errMsg, name));
     }
+    tableMap.put(name, table);
+  }
+
+  @Override
+  public String getEstimatedKeyCount() {
```

Comment:
I typed this earlier, but forgot to post it. This means we are going to have a metric where the value is a command seperated list of key value pairs, eg:

```
metric -> tab1 : 10, tab2 : 15 ... etc.
```
Have we got any other metrics like this? Would it be better if we had a metric for each table directly, so they can be charted etc if needed? Ie

```
SCMEstimatedKeyCountTab1 -> 10
SCMEstimatedKeyCounttab2 -> 15
etc
```

You would need to have dynamic metric names for this, but there is an example of doing that in ReplicationManagerMetrics

Reply:
@sodonnel Thank you for the advice. Updated the PR, please have a look.